config/cscs.py

```diff
@@ -205,28 +207,66 @@ class ReframeSettings:
                 },
                 'PrgEnv-pgi': {
                     'type': 'ProgEnvironment',
-                    'modules': ['PrgEnv-pgi/17.10'],
+                    'modules': ['PrgEnv-pgi/17.10_gdr'],
                     'cc': 'mpicc',
                     'cxx': 'mpicxx',
                     'ftn': 'mpif90',
                 },
-                'PrgEnv-pgi_17.10_gdr': {
+                'PrgEnv-c2sm-gnu': {
                     'type': 'ProgEnvironment',
-                    'modules': ['PrgEnv-pgi/17.10_gdr'],
+                    'modules': ['PrgEnv-gnu', '/apps/common/UES/sandbox/kraushm/c2sm-rcm-env/env', 'c2sm/gnu-env/base'],
+                    'cc': 'mpicc',
+                    'cxx': 'mpicxx',
+                    'ftn': 'mpif90',
+                },
+                'PrgEnv-c2sm-gnu-gpu': {
+                    'type': 'ProgEnvironment',
+                    'modules': ['PrgEnv-gnu', '/apps/common/UES/sandbox/kraushm/c2sm-rcm-env/env', 'c2sm/gnu-env/gpu'],
```

Comment:
For

'modules': ['PrgEnv-gnu', '/scratch/ajocksch/c2sm-rcm-env.git/modules/kesch/c2sm/gnu-env/gpu', 'mvapich2gdr_gnu/2.2_cuda_8.0'],

it is working

Reply:
> btw. why is 'PrgEnv-gnu' needed?

I think we should focus only on the Cray and PGI version for the moment.